libs/contract/src/lib/avails/avails.ts

```diff
@@ -90,6 +90,23 @@ function getMatchingSales<T extends (FullSale | BucketContract)>(sales: T[], ava
   }));
 }
 
+export function getMandateTerms(avails: AvailsFilter, terms: Term<Date>[]): Term<Date>[] | undefined {
+  const result = terms.filter(term =>
+    allOf(avails.duration).in(term.duration)
+    && allOf(avails.medias).in(term.medias)
+    && allOf(avails.territories, 'optional').in(term.territories)
+  );
```

Comment:
Please use a scope in a callback with more than one line : 
```suggestion
  const result = terms.filter(term => {
    return allOf(avails.duration).in(term.duration)
    && allOf(avails.medias).in(term.medias)
    && allOf(avails.territories, 'optional').in(term.territories)
  });
```